### DISCUSSION TITLE
Capture the Wool wins missing

### DISCUSSION THREAD

**Discussion by daethy** [HypixelDev/PublicAPI#577] [Q&A]
Hi,

I've been working on a bot and I would like to return the number of total arcade wins for a player. As total arcade wins aren't tracked in the public API, I tried to add up the wins of every arcade game. Unfortunately, Capture the Wool wins are not available in the public API making obtaining the total arcade wins impossible. This is the only arcade game which wins aren't tracked in the API.

I have considered using the `player?.achievements?.arcadeArcadeWinner` key, however, this is extremely inaccurate as it does not include wins gained in prototype.

**Comment by ConnorLinfoot** [comment]
The API exposes data from our player database. Not all data is there due to legacy implementations of systems or other various reasons, it could be moved one day but it's not something that is actively being looked at nor an issue with the API itself.
